Simplenote/TagListViewController+Swift.swift

```diff
@@ -282,3 +286,17 @@ extension TagListViewController: SPTextFieldDelegate {
         !menuShowing
     }
 }
+
+
+// MARK: - Appearance
+//
+extension TagListViewController {
+    @objc
+    func refreshTableRowsActiveStatus() {
+        for row in 0..<tableView.numberOfRows {
+            let tableRow = tableView.rowView(atRow: row, makeIfNecessary: false) as? TableRowView
+            tableRow?.isActive = isActive
+        }
```

Comment:
Q: do you think it'd be worthy to move this into `NSTableView+Simplenote` (and deduplicate?)

Reply:
Yes, done! Thanks!